<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1201</width>
      <height>681</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="realTimeTab">
     <attribute name="title">
      <string>Real Time Plots</string>
     </attribute>
     <widget class="QFrame" name="pyranometerFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>150</width>
        <height>260</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pyranometerStopButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop Display</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="pyranometerLcd">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pyranometerStartButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Display</string>
       </property>
      </widget>
      <widget class="QLabel" name="pyranometerLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Solar Irradiance</string>
       </property>
      </widget>
      <widget class="QLabel" name="pyranometerUnit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>W/m2</string>
       </property>
      </widget>
     </widget>
     <widget class="QCustomPlot" name="OPLSPlot" native="true">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>380</y>
        <width>361</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>629</x>
        <y>99</y>
        <width>151</width>
        <height>261</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>380</y>
        <width>151</width>
        <height>261</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QCustomPlot" name="lidarPlot" native="true">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>430</width>
        <height>260</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="windPlot" native="true">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>100</y>
        <width>361</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="lidarFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>150</width>
        <height>260</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLCDNumber" name="lidarLcd">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lidarLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Lidar distance</string>
       </property>
      </widget>
      <widget class="QPushButton" name="lidarStartButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Display</string>
       </property>
      </widget>
      <widget class="QPushButton" name="lidarStopButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop Display</string>
       </property>
      </widget>
      <widget class="QLabel" name="lidarUnit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>cm</string>
       </property>
      </widget>
     </widget>
     <widget class="QCustomPlot" name="pyranometerPlot" native="true">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>380</y>
        <width>430</width>
        <height>260</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="startRecordButton">
      <property name="geometry">
       <rect>
        <x>138</x>
        <y>30</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Record all data</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopRecordButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Record all data</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="recordTab">
     <attribute name="title">
      <string>Recorded Plot</string>
     </attribute>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>19</y>
        <width>151</width>
        <height>551</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="loadFileButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Load ...</string>
       </property>
      </widget>
     </widget>
     <widget class="QCustomPlot" name="loadedPlot" native="true">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>1011</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
     <zorder>loadedPlot</zorder>
     <zorder>frame_5</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1236</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
